base/docs/helpdb/Base.jl

```diff
@@ -2265,11 +2265,11 @@ sortperm
 """
     mod2pi(x)
 
-Modulus after division by 2pi, returning in the range \[0,2pi).
+Modulus after division by ``2π``, returning in the range ``[0,2π)``.
```

Comment:
should pick and be consistent whether these should be highlighted as code (single backticks in markdown) or as latex-style math (double backticks in our markdown flavor)